go/cmd/vtctldclient/command/schema.go

```diff
@@ -90,6 +98,25 @@ For --sql, semi-colons and repeated values may be mixed, for example:
 		Args:                  cobra.ExactArgs(1),
 		RunE:                  commandReloadSchemaShard,
 	}
+	// ValidateSchemaKeyspace makes a ValidateSchemaKeyspace gRPC call to a vtctld.
+	ValidateSchemaKeyspace = &cobra.Command{
+		Use:                   "ValidateSchemaKeyspace [--exclude-tables=<exclude_tables>] [--include-views] [--skip-no-primary] [--include-vschema] <keyspace>",
+		Short:                 "Validates that the schema on the primary tablet for the first shard matches the schema on all other tablets in the keyspace.",
```

Comment:
```suggestion
		Short:                 "Validates that the schema on the primary tablet for the first shard matches the schema on all other shards in the keyspace.",
```

Reply:
I copied the text from the legacy client, and the server code actually does compare against all tablets.

Reply:
ok :)